todo_app/templates/index.html

```diff
@@ -27,12 +31,59 @@ <h2>Items</h2>
           </tr>
         </thead>
         <tbody>
-          <!-- List the todo items here -->
-          {% for item in items %}
-          <tr>
+          <!-- List To Do items here -->
+          {% for item in view_model.todo_items %}
+          <tr data-status="{{ item.status }}">
             <td>{{ item.name }}</td>
             <td>{{ item.status }}</td>
             <td>{{ item.description }}</td>
+              <!-- Action buttons for To Do items -->
+              <td>
+                <form action="/mark_complete/{{item.id}}" method="post" class="form-example">
+                  <input type="hidden" name="id" value="{{ item.id }}">
+                  {% if item.status == "To do" %}
+                    <input type="submit" class="btn btn-success" name="mark_done"value="Complete">
+                  </form>
+                    <form action="/mark_progress/{{item.id}}" method="post" class="form-example">
+                    <input type="submit" class="btn btn-info" name="in_progress" value="In Progress">
+                  </form>
+                  {% elif item.status == "In Progress" %}
+                    <input type="submit" class="btn btn-success" value="Complete">
+                  {% elif item.status == "Done" %}
+                    <input type="submit" class="btn btn-secondary" name="undo" value="Done" disabled>
+                  {% endif %}
```

Comment:
Can we significantly simplify these if-chains now, since we already know the items in this section are all To-Do, and the next duplicate block is all "In Progress"?